deploy/archive-sync-ols-copy.yaml

```diff
@@ -0,0 +1,282 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: archive-sync-ols-copy
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: archive-sync-ols-copy
+  spec:
+    envName: ${ENV_NAME}
+    objectStore:
+      - ${TARGET_S3_BUCKET}
+    testing:
+      iqePlugin: ccx
+    deployments:
+      - name: instance
+        replicas: ${{REPLICAS}}
+        webServices:
+          public:
+            enabled: false
+          private:
+            enabled: false
+          metrics:
+            enabled: true
+        podSpec:
+          image: ${IMAGE}:${IMAGE_TAG}
+          command: ["ccx-messaging"]
+          args: ["/data/config.yaml"]
+          env:
+            - name: CLOWDER_ENABLED
+              value: ${CLOWDER_ENABLED}
+            - name: SENTRY_DSN
+              valueFrom:
+                secretKeyRef:
+                  key: dsn
+                  name: archive-sync-ols-dsn
+                  optional: true
```

Comment:
Do we want to send the sentry events from 2 different services to the same Glitchtip project?